go/vt/binlog/event_streamer.go

```diff
@@ -232,10 +232,10 @@ func parsePkTuple(tokenizer *sqlparser.Tokenizer, insertid int64, fields []mprot
 
 			// update type
 			switch fields[index].Type {
-			case mproto.VT_DECIMAL:
+			case sqltypes.Null:
 				// we haven't updated the type yet
-				fields[index].Type = mproto.VT_LONGLONG
-			case mproto.VT_LONGLONG:
+				fields[index].Type = sqltypes.Int64
```

Comment:
Never mind this comment, this is wrong in the code, but I don't think this is the time to fix this.